docs/user_guide/sysnand/sending_payload.md

```diff
@@ -79,6 +79,26 @@ Now that the device is in RCM, we will need to send it a payload. The methods ar
 
 &nbsp;
 
+## Chomebooks
+
+### What you need
+
+!!! tip ""
+    - The latest release of <a href="https://github.com/CTCaer/hekate/releases/" target="_blank">Hekate</a> (either the hekate_ctcaer bin or the hekate_ctcaer zip)
+    - A USB-C cable
+    - If your chromebook has a USB-C note that this will not work using a C-C cable
+	
+### Instructions
+    
+!!! tip ""
+    1. Go to the website https://switchgui.de/web-payload/ and scroll all the way down
```

Comment:
You are aware you can use `[this](https://switchgui.de/web-payload/)` to show links, right? like [this](https://switchgui.de/web-payload/)